A.css

```diff
@@ -1,3 +1,3 @@
 h1{
-    color: black;

```

Comment:
Since you do not have any commits on this branch, I will review your branch locally then list all the changes you can make here:
***Please make sure that you only work on this branch to apply the changes. DO NOT MERGE IT before this branch is approved. Kindly submit the same Pull Request for a re-review when you are done pushing all your changes***

`.gitignore` file:
![image](https://user-images.githubusercontent.com/38088818/164088526-7199002f-7994-4aaf-8fc7-0245b20976de.png)

-  Your .gitignore file should only have `test.md` and `node_modules` listed in it for now. You do not need to have all the other files.

Try to fix the following Stylint errors so that all your tests pass:
![image](https://user-images.githubusercontent.com/38088818/164089229-910871ec-2f10-4eca-9c56-454fb78c0ff5.png)